src/entities/IDealTypes.ts

```diff
@@ -1,15 +1,12 @@
 import { IDealRegistrationTokenSwap } from "./DealRegistrationTokenSwap";
 
-export interface IVoteInfo {
-  address: string;
-  vote: boolean;
-}
+export type IVotesInfo = Record<string, boolean>;
```

Comment:
votes can also be `null` when a representative didn't yet vote, right?

Reply:
Yes, should be update the type?

Reply:
yes please. It will help us when building some logic based on those

Reply:
I am not clear what is the issue here.  Has it been addressed?  @IonelLupu

Reply:
@dkent600  since @paweljanicki  is off, I fixed the type.

The issues was the type of the vote which is `boolean | null` in reality and not just `boolean`. I pushed a commit with the fix